packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -55,82 +52,89 @@ export const PaymentConfirmation = ({
   );
 
   return (
-    <section
-      className={`container card payment-confirmation ${className}`}
-      data-testid="payment-confirmation"
-    >
-      <header>
-        <img
-          className="circled-check"
-          src={circledCheckbox}
-          alt="circled checkbox"
-        />
-        {heading}
-        <Localized id="payment-confirmation-subheading">
-          <p></p>
-        </Localized>
-        <p>{email}</p>
-      </header>
-
-      <div className="order-details">
-        <Localized id="payment-confirmation-order-heading">
-          <h3></h3>
-        </Localized>
-        <div className="bottom-row">
+    <>
+      <SubscriptionTitle screenType="success" />
+      <section
+        className={`container card payment-confirmation ${className}`}
+        data-testid="payment-confirmation"
+      >
+        <header>
+          <img
+            className="circled-check"
+            src={circledCheckbox}
+            alt="circled checkbox"
+          />
+          <Localized id="payment-confirmation-thanks-heading">
+            <h2>Thank you!</h2>
+          </Localized>
           <Localized
-            id="payment-confirmation-invoice-number"
-            vars={{ invoiceNumber }}
+            id="payment-confirmation-thanks-subheading"
+            vars={{ email, product_name }}
           >
-            <p></p>
+            <p>{`A confirmation email has been sent to ${email} with details on how to get started with ${product_name}.`}</p>
           </Localized>
-          <p>{date}</p>
-        </div>
-      </div>
+        </header>
 
-      {Provider.isStripe(payment_provider) && (
-        <div className="billing-info" data-testid="billing-info">
-          <Localized id="payment-confirmation-billing-heading">
+        <div className="order-details">
+          <Localized id="payment-confirmation-order-heading">
             <h3></h3>
           </Localized>
           <div className="bottom-row">
-            {displayName ? <p>{displayName}</p> : null}
-            <p>{email}</p>
+            <Localized
+              id="payment-confirmation-invoice-number"
+              vars={{ invoiceNumber }}
+            >
+              <p></p>
+            </Localized>
+            <p>{date}</p>
           </div>
         </div>
-      )}
 
-      <div className="payment-details">
-        <Localized id="payment-confirmation-details-heading">
-          <h3></h3>
-        </Localized>
-        <div className="bottom-row">
-          <Localized
-            id={`payment-confirmation-amount-${interval}`}
-            vars={{
-              amount: getLocalizedCurrency(amount, currency),
-              intervalCount: interval_count,
-            }}
-          >
-            <p>{planPrice}</p>
+        {Provider.isStripe(payment_provider) && (
+          <div className="billing-info" data-testid="billing-info">
+            <Localized id="payment-confirmation-billing-heading">
+              <h3></h3>
```

Comment:
Fallback content?

Reply:
Interesting -- this actually isn't something I touched in my patch; my local development environment is reformatting this file and committing the reformat... I can update it though...